tests/integration/cairo1_hint_locals/prepare/prepare_test.cairo

```diff
@@ -0,0 +1,21 @@
+use array::ArrayTrait;
+use result::ResultTrait;
+
+#[test]
+fn test_prepare_no_args() {
+    let class_hash = declare('minimal_no_args').unwrap();
+
+    let calldata = ArrayTrait::new();
+    let prepared_contract = prepare(class_hash, calldata).unwrap();
+    drop(prepared_contract);
+}
+
+#[test]
+fn test_prepare_with_args() {
+    let class_hash = declare('minimal_with_args').unwrap();
+
+    let mut calldata = ArrayTrait::new();
+    calldata.append('name');
```

Comment:
Can we include few more arguments here? Possibly with more complex types. I believe cairo1 supports something like `into` which lets user dump the data.

Reply:
Will add more complex test in a next PR